keps/sig-node/753-sidecar-containers/README.md

```diff
@@ -938,10 +1116,37 @@ Update OOM score adjustment:
 
 ##### Integration tests
 
-The feature will be covered with unit tests and e2e tests.
+<!--
+Integration tests are contained in k8s.io/kubernetes/test/integration.
+Integration tests allow control of the configuration parameters used to start the binaries under test.
+This is different from e2e tests which do not allow configuration of parameters.
+Doing this allows testing non-default options and multiple different and potentially conflicting command line options.
+-->
+
+<!--
+This question should be filled when targeting a release.
+For Alpha, describe what tests will be added to ensure proper quality of the enhancement.
+
+For Beta and GA, add links to added tests together with links to k8s-triage for those tests:
+https://storage.googleapis.com/k8s-triage/index.html
+-->
+
+- <test>: <link to test coverage> FIXME to fill
 
 ##### e2e tests
 
+<!--
+This question should be filled when targeting a release.
+For Alpha, describe what tests will be added to ensure proper quality of the enhancement.
+
+For Beta and GA, add links to added tests together with links to k8s-triage for those tests:
+https://storage.googleapis.com/k8s-triage/index.html
+
+We expect no non-infra related flakes in the last month as a GA graduation criteria.
+-->
+
+- <test>: <link to test coverage> FIXME to fill
```

Comment:
@tzneal @gjkim42 here too?

Reply:
adding test lists and their failure rate link to each subsection should be enough.

`ginkgo run -v --dry-run --focus=SidecarContainers ./test/e2e_node/`
this command should be helpful.
```
$ ginkgo run -v --dry-run --focus=SidecarContainers ./test/e2e_node/
...
[sig-node] [NodeAlphaFeature:SidecarContainers] Containers Lifecycle  when using a restartable init container in a Pod with restartPolicy=Always when a restartable init container runs continuously should 
keep running a Pod continuously and produce log                                                                                                                                                             
/home/gjkim/go/src/k8s.io/kubernetes/test/e2e_node/container_lifecycle_test.go:1889                                                                                                                         
• [0.000 seconds]                                                                                                                                                                                           
------------------------------                                                                                                                                                                              
[sig-node] [NodeAlphaFeature:SidecarContainers] Containers Lifecycle  when using a restartable init container in a Pod with restartPolicy=Always when a restartable init container runs continuously should 
not restart a restartable init container                                                                                                                                                                    
/home/gjkim/go/src/k8s.io/kubernetes/test/e2e_node/container_lifecycle_test.go:1901                                                                                                                         
• [0.000 seconds]                                                                                                                                                                                           
------------------------------                                                                                                                                                                              
[sig-node] [NodeAlphaFeature:SidecarContainers] Containers Lifecycle  when using a restartable init container in a Pod with restartPolicy=Always when a restartable init container runs continuously should 
start a regular container                                                                                                                                                                                   
/home/gjkim/go/src/k8s.io/kubernetes/test/e2e_node/container_lifecycle_test.go:1905                                                                                                                         
• [0.000 seconds]                                                                                                                                                                                           
------------------------------                                                                                                                                                                              
[sig-node] [NodeAlphaFeature:SidecarContainers] Containers Lifecycle  when using a restartable init container in a Pod with restartPolicy=Always when a restartable init container fails to start because of
 a bad image should continuously run Pod keeping it Pending and produce log                                                                                                                                 
/home/gjkim/go/src/k8s.io/kubernetes/test/e2e_node/container_lifecycle_test.go:1948                                                                                                                         
• [0.000 seconds]                                                                                                                                                                                           
------------------------------                                                                                                                                                                              
[sig-node] [NodeAlphaFeature:SidecarContainers] Containers Lifecycle  when using a restartable init container in a Pod with restartPolicy=Always when a restartable init container fails to start because of
 a bad image should not start a restartable init container                                                                                                                                                  
/home/gjkim/go/src/k8s.io/kubernetes/test/e2e_node/container_lifecycle_test.go:1960                                                                                                                         
• [0.000 seconds]                                                                                                                                                                                           
------------------------------                                                                                                                                                                              
[sig-node] [NodeAlphaFeature:SidecarContainers] Containers Lifecycle  when using a restartable init container in a Pod with restartPolicy=Always when a restartable init container fails to start because of
 a bad image should not start a regular container                                                                                                                                                           
/home/gjkim/go/src/k8s.io/kubernetes/test/e2e_node/container_lifecycle_test.go:1963                                                                                                                         
• [0.000 seconds]                                                                                                                                                                                           
------------------------------
...
```